### ISSUE TITLE
[Bug]: Storybook 9 canary - Coverage button is clickable even though there is no coverage

### ISSUE THREAD

**Issue by yannbf** [storybookjs/storybook#30884]
### Describe the bug

Seems like the coverage button should be disabled, similar to how it is in the a11y dot
<img width="384" alt="Image" src="https://github.com/user-attachments/assets/b1043f58-63c3-4e27-b3fc-9a2efea651e7" />

### Reproduction link

-

### System

```bash
-
```
